app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
+    if len(command.split()) != 3:
+        raise ValueError("command should have 3 arguments")
+    src_file_path = command[1]
+    dst_file_path = command[2]
```

Comment:
command[1] is "p" from the word cp
etc.
create a separated variable for command.split() and call it on demand